DataFormats/PortableMath/interface/deltaPhi.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DataFormats_PortableMath_deltaPhi_h
```

Comment:
should it just be in `DataFormats/Math/interface/alpaka/deltaPhi.h` ?

The file is missing some basic alpaka-related headers to pass the header check (it should compile by itself)

Reply:
I explained the logic for the new subpackage in the description but I have no real preference, so I will go by your suggestion. In any case, we will get feedback during the PR review.

Reply:
> I will check for additional headers.

Working back for this, I see that `LSTCore/src/alpaka/Hit.h`, which includes these functions, includes
https://github.com/SegmentLinking/cmssw/blob/3950ee4bb3aa69cbbffe9948322d8a4c138e99a7/RecoTracker/LSTCore/src/alpaka/Hit.h#L4-L6

Ignoring the SoA related headers, which do not have non-SoA related dependencies, the only relevant inclusion in `LSTCore/interface/alpaka/Common.h` is
https://github.com/SegmentLinking/cmssw/blob/3950ee4bb3aa69cbbffe9948322d8a4c138e99a7/RecoTracker/LSTCore/interface/alpaka/Common.h#L7
and, from that file,
https://github.com/SegmentLinking/cmssw/blob/3950ee4bb3aa69cbbffe9948322d8a4c138e99a7/RecoTracker/LSTCore/interface/Common.h#L5
That should be enough to cover the headers but let me know otherwise, or if something more targeted should be used (I partially hope that I will be pointed to that during review, if that's the case).

Reply:
I had in mind just the alpaka basic header `#include <alpaka/alpaka.hpp>` (not sure it brings in the math functions). But I guess the `HeterogeneousCore/AlpakaInterface/interface/config.h` can work as well

Reply:
> should it just be in `DataFormats/Math/interface/alpaka/deltaPhi.h` ?

I see that it was instead moved to `DataFormats/Math/interface/deltaPhi.h`

Reply:
Oops, sorry, I misread your initial comment. I will move it there.